examples/fuzzy_deduplication.py

```diff
@@ -77,26 +79,20 @@ def main(args):
             false_positive_check=False,
         )
         fuzzy_dup = FuzzyDuplicates(logger=log_dir, config=fuzzy_dedup_config)
-        duplicates = fuzzy_dup(dataset=input_dataset)
+
+        # When perform_removal=False, it'll only call .identify_duplicates() and return the duplicates.
+        # When perform_removal=True then exact_dup outputs dataset with the duplicates removed
+        # It'll behave by calling .identify_duplicates() and .removal() in sequence.
```

Comment:
```suggestion
        # When perform_removal=True, then exact_dup outputs the dataset with the duplicates removed.
        # It will behave by calling .identify_duplicates() and .remove() in sequence.
```